drivers/hmis/lib/tasks/one_time_migration_20240303.rake

```diff
@@ -15,6 +15,26 @@ class OneTimeMigration20230303
     update_form_definitions
   end
 
+  # Migrate LinkIDs for all files in drivers/hmis/lib/form_data. Write changes to files.
+  # Doesn't catch everything, a few things need to be manually fixed (client-id and veteran-status)
+  def update_form_data_files
```

Comment:
this isn't called from perform, we'd just call this manually and commit the changes